padacioso/bracket_expansion.py

```
@@ -214,5 +214,54 @@ def translate_padatious(example: str) -> str:
+
+def normalize_apostrophes(text: str) -> str:
+    """
+    Normalize various apostrophe/quote unicode variants to a plain ASCII apostrophe.
+    Handles curly apostrophes, typographic quotes, and similar characters that
+    look like apostrophes but are different code points.
+    @param text: input text
+    @return: text with normalized apostrophes
+    """
+    # Right single quotation mark, modifier letter apostrophe, heavy apostrophe, etc.
+    apostrophe_variants = [
+        '’',  # RIGHT SINGLE QUOTATION MARK
```

Ruff (0.15.10)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)